animareflection/ui
# E2E Testing Patterns

## Description

##### Task link: https://trello.com/c/J66ahrAR/154-e2e-interaction-testing

Added pattern / tests for interactive components using `@storybook/testing-library` addon.

## Test Steps

1) Verify that testing pattern looks appropriate.
2) Verify that tests pass.
3) If additional tests or modifications should be made, let me know.
